tests/internal/test/service.go

```diff
@@ -66,7 +66,14 @@ func GetServices(t *testing.T, cfg *configs.Config) Services {
 
 	km := basic.NewKeyManager(cfg, keyStore, fc)
 
-	wp := jobs.NewWorkerPool(jobStore, 100, 1)
+	wp := jobs.NewWorkerPool(
+		jobStore, 100, 1,
```

Comment:
This could utilize `cfg.WorkerCount`  and maybe also `cfg.WorkerQueueCapacity` - thoughts?